File: Java/Program/Product.java
public class Product {
    
    private int idProduct;
    private String name;
    private String brand;
    private int price;
    
    public Product(int idProduct, String name, String brand, int price) {
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }
    
    public int getIdProduct() {
        return idProduct;
    }
    public void setIdProduct(int idProduct) {
        this.idProduct = idProduct;
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public String getBrand() {
        return brand;
    }
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public int getPrice() {
        return price;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    public void showInfo() {
        System.out.println("ID Product: " + idProduct);
        System.out.println("Name: " + name);
        System.out.println("Brand: " + brand);
        System.out.println("Price: " + price);
    }
}

File: CPP/Program/Product.cpp
#include <string>

using namespace std;

class Product
{
    private:
        int idProduct;
        string name;
        string brand;
        int price;

    public:
        /* Constructor */
        Product()
        {
            this->idProduct = 0;
            this->name = "";
            this->brand = "";
            this->price = 0;
        }

        Product(int idProduct, string name, string brand, int price)
        {
            this->idProduct = idProduct;
            this->name = name;
            this->brand = brand;
            this->price = price;
        }

        int getIdProduct() const
        {
            return idProduct;
        }
        void setIdProduct(int idProduct)
        {
            this->idProduct = idProduct;
        }

        string getName() const
        {
            return name;
        }
        void setName(string name)
        {
            this->name = name;
        }

        string getBrand() const
        {
            return brand;
        }
        void setBrand(string brand)
        {
            this->brand = brand;
        }

        int getPrice() const
        {
            return price;
        }
        void setPrice(int price)
        {
            this->price = price;
        }

        /* Destructor */
        ~Product()
        {
            
        }
};
